src/main.rs

```diff
@@ -118,6 +118,14 @@ enum Expr {
     },
 }
 
+/// [RustTypeKind] represents the different kinds of rust types: [struct]s,
+/// [enum]s, aliases...
+enum RustTypeKind {
+    RustStruct,
+    RustEnum,
+    RustAlias,
```

Comment:
```suggestion
    Struct,
    Enum,
    Alias,
```
as this is an AST in-between Coq and Rust.